vivbin

```diff
@@ -63,7 +63,12 @@ def main():
             verbose = True
 
         elif opt == "-O":
-            vw.config.parseConfigOption( optarg )
+            try:
+                vw.config.parseConfigOption( optarg )
+            except:
+                print vw.config.reprConfigPaths(True)
```

Comment:
heh.  also, probably wanna print the "humon readble" line after the traceback if we're gonna keep it.